tests/com/twitter/intellij/pants/util/PantsUtilTest.java

```diff
@@ -24,19 +27,29 @@ public void testIsPantsProjectFile() {
   }
 
   public void testFindJdk() {
-    Optional<File> executable = PantsUtil.findPantsExecutable(getProjectFolder());
-    assertTrue(executable.isPresent());
-    Optional<Sdk> sdkA = PantsUtil.getDefaultJavaSdk(executable.get().getPath());
-    assertTrue(sdkA.isPresent());
-    ApplicationManager.getApplication().runWriteAction(new Runnable() {
-      @Override
-      public void run() {
-        ProjectJdkTable.getInstance().addJdk(sdkA.get());
-      }
+    final File executable = PantsUtil.findPantsExecutable(getProjectFolder()).get();
+    final Sdk sdkA = getDefaultJavaSdk(executable.getPath()).get();
+    List<Sdk> sdkResults = Arrays.stream(ProjectJdkTable.getInstance().getAllJdks())
+      .filter(jdk -> jdk == sdkA)
+      .collect(Collectors.toList());
+    assertTrue(sdkResults.size() == 1);
+    assertTrue(sdkResults.get(0) == sdkA);
```

Comment:
`assertSame` same for below

Reply:
I think using `assertEqual` would do the same check as `assertSame` for each element of the two collections, so we don't need to use it if we're using `assertEqual` to check the identity of each element in the list. Am I missing something?